catroid/src/main/res/values/strings.xml

```diff
@@ -1577,5 +1577,8 @@
     <string name="crash_dialog_always_send">Always send</string>
     <string name="crash_dialog_send">Send</string>
     <string name="plus" translatable="false">+</string>
+
+    <!--Solution to handle fabric's UnmetDependencyException during the test -->
+    <string name="com.crashlytics.android.build_id" translatable="false">RANDOM_UUID</string>
```

Comment:
Once the fabric.properties file is moved to the catroid folder, remove this string.

```
./gradlew clean check
…
* What went wrong:
Execution failed for task ':catroid:mergeCatroidReleaseResources'.
[string/com.crashlytics.android.build_id] …/catroid/src/main/res/values/strings.xml
[string/com.crashlytics.android.build_id] …/catroid/build/generated/fabric/res/catroid/release/values/com_crashlytics_build_id.xml:
Error: Duplicate resources
…
```